ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Network/NodeToNode.hs

```diff
@@ -199,6 +203,15 @@ data Handlers m addr blk = Handlers
       NodeToNodeVersion ->
       ConnectionId addr ->
       TxSubmissionServerPipelined (GenTxId blk) (GenTx blk) m ()
+  , hPerasCertDiffusionInbound ::
```

Comment:
Add a TODO here that this actually needs to be passed to the Network layer? Ie https://github.com/tweag/cardano-peras/issues/78